docs/io/unified.rst

```diff
@@ -281,9 +281,8 @@ column use:
      >>> dat = ascii.read('file.dat', format='daophot')
      >>> dat = Table.read('file.dat', format='ascii.daophot')
 
-   For compatibility with ``astropy`` version 0.2 and earlier, the following
-   format values are also allowed in ``Table.read()``: ``daophot``, ``ipac``,
-   ``html``, ``latex``, and ``rdb``.
+   The following format values are also allowed in ``Table.read()``:
```

Comment:
@taldcroft - one specific item perhaps for you to consider: should we just remove this mentioning of the short-cuts?

Reply:
The `doctest-skip` directive on line 279 breaks the formatting here, and fixing that by indenting the directive breaks the tests. That means a more substantial rewrite is definitely required but I'm not sure this PR is the best place for that.

Reply:
Yes, 👍 on removing reference to the shortcuts.

Reply:
Removing those lines does help with the formatting, but doesn't fix it entirely.